README.md

```diff
@@ -0,0 +1,91 @@
+Audio Visualizer using DragonBoard410c
```

Comment:
Well, the title should be highlighted so 

#Audio Visualizer using DragonBoard410c